### ISSUE TITLE
Validate no special characters (ie. @) for GitHub handles

### ISSUE THREAD

**Issue by binarymason** [charlottejuniordevs/api#11]
Need to add this line:

```rb
  validates :github_handle, :format => { with: /\A[a-zA-Z0-9]*\z/ , :message => 'no special characters, only letters and numbers' }

```
here:
https://github.com/charlottejuniordevs/api/blob/development/app/models/user.rb#L27

**Comment by TammyTee** [comment]
👩🏾‍💻☝🏾

**Comment by kirillian** [comment]
This issue appears to be solved by #16
